src/modules/transaction/utils/encoding.js

```diff
@@ -124,17 +123,17 @@ export const fromTransactionJSON = (transactionJSON, paramsSchema) => {
 export const toTransactionJSON = (transaction, paramsSchema) => {
   if (Buffer.isBuffer(transaction.params)) {
     return {
-      ...codec.toJSON(baseTransactionSchema, transaction),
-      params: paramsSchema ? codec.decodeJSON(paramsSchema, transaction.params) : {},
+      ...liskCodec.codec.toJSON(baseTransactionSchema, transaction),
```

Comment:
do we have ticket for liskCodec changes? if not please create one and assign to the pr